src/pages/interest/Interest.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import useDocumentTitle from "../../hooks/useDocumentTitle";
+import useTrackPageview from "../../hooks/useTrackPageview";
+import { CONTEXT_PATH } from "../../environment";
+import useScrollToTop from "../../hooks/useScrollToTop";
+import BackLink from "../../components/backlink/BackLink";
+import H1WithAutoFocus from "../../components/h1WithAutoFocus/H1WithAutoFocus";
+import "./Interest.less";
+import "../../components/buttons/LinkMainButton.less";
+
+function Interest() {
+    const [stillingId, setStillingId] = useState(null);
+    useEffect(() => {
+        setStillingId(document.location.pathname.split("/")[3]);
+    }, []);
```

Comment:
Denne siden er routet til i app.js. Routeren sender diverse relaterte routing-props til hvert view, og du kan hente ut `uuid` fra `match.params.uuid`. 

Du trenger nok ikke useState og useEffect for å holde på stillingsId. Tror dette burde holde:
```
function Interest({match}) {
    const stillingId = match.params.uuid
    ... og at useState og useEffect droppes
   ```

Reply:
ooh nice